lms/djangoapps/verify_student/views.py

```diff
@@ -1105,7 +1105,7 @@ def _send_confirmation_email(self, user):
 
         subject = _("Verification photos received")
         message = render_to_string('emails/photo_submission_confirmation.txt', context)
-        from_address = configuration_helpers.get_value('default_from_email', settings.DEFAULT_FROM_EMAIL)
+        from_address = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
```

Comment:
@felipemontoya I totally understand, but don't we have this same problem with this change?